packages/slider/test/slider.test.ts

```diff
@@ -1604,4 +1607,128 @@ describe('Slider', () => {
     });
+    it('resets to default value on double click after moving pointer', async () => {
+        const el = await fixture<Slider>(
+            html`
+                <sp-slider
+                    style="width: 100px"
+                    value="50"
+                    default-value="50"
+                ></sp-slider>
+            `
+        );
+        await elementUpdated(el);
+        expect(el.value, 'initial').to.equal(50);
+
+        const handle = el.shadowRoot.querySelector('.handle') as HTMLDivElement;
+        const handleBoundingRect = handle.getBoundingClientRect();
+        const position: [number, number] = [
+            handleBoundingRect.x + handleBoundingRect.width / 2,
+            handleBoundingRect.y + handleBoundingRect.height / 2,
+        ];
+        await sendMouse({
+            steps: [
+                {
+                    type: 'move',
+                    position,
+                },
+                {
+                    type: 'down',
+                },
+            ],
+        });
+
+        await elementUpdated(el);
+        await sendMouse({
+            steps: [
+                {
+                    type: 'move',
+                    position: [
+                        150,
+                        handleBoundingRect.y + handleBoundingRect.height + 100,
+                    ],
+                },
+                {
+                    type: 'up',
+                },
+            ],
+        });
+
+        await elementUpdated(el);
+
+        // since we've moved the pointer, the new value should be 100
+        expect(el.value).to.equal(100);
+
+        handle.dispatchEvent(
+            new PointerEvent('dblclick', {
+                clientX: 0,
+                cancelable: true,
+                button: 0,
+                composed: true,
+                bubbles: true,
+            })
+        );
+
+        await elementUpdated(el);
+
+        expect(
+            el.value,
+            'reset to default value on double click after moving pointer'
+        ).to.equal(50);
+    });
+    it('manages escape key interactions correctly in an overlaid context', async () => {
+        const el = await fixture<HTMLDivElement>(
+            html`
+                <div>
+                    <sp-button id="trigger">Overlay Trigger</sp-button>
+                    <sp-overlay trigger="trigger@click" placement="bottom">
+                        <sp-popover>
+                            <sp-slider
+                                style="width: 100px"
+                                value="70"
+                                default-value="50"
+                            ></sp-slider>
+                        </sp-popover>
+                    </sp-overlay>
+                </div>
+            `
+        );
+
+        await elementUpdated(el);
+
+        // open the overlay
+        const trigger = el.querySelector('#trigger') as HTMLButtonElement;
+        trigger.click();
```

Comment:
Overlay open action should follow the pattern:

```
const opened = oneEvent(el, 'sp-opened');
trigger.click(); // or similiar
await opened;
```
Otherwise there's no guarantee the content actually opened and was promoted to the top layer.

Reply:
got it!